.github/workflows/dash-bmv2-ci.yml

```diff
@@ -54,8 +54,6 @@ jobs:
       run:  make docker-dash-p4c
     - name: Build P4 software switch (bmv2) and P4Info
       run:  DOCKER_FLAGS=$docker_fg_flags make p4
-    - name: Install DASH submodules recursively
```

Comment:
I've made the same change in two other CI files. I believe that should be reverted as well.

Reply:
Please back the changes out and we'll verify one more time with a clean local build and in CI (which is similar to a clean local build).